compiler/x/codegen/OMRCodeGenerator.hpp

```diff
@@ -594,7 +594,8 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
 
    bool nodeIsFoldableMemOperand(TR::Node *node, TR::Node *parent, TR_RegisterPressureState *state);
 
-   TR::IA32ConstantDataSnippet     *findOrCreateConstant(TR::Node *, void *c, uint8_t size);
+   TR::IA32DataSnippet         *createData(TR::Node *, void *c, uint8_t size);
```

Comment:
Doxygen comments for these two functions please.

Reply:
Updated.